### PR TITLE
2757 cli 2

### PR DESCRIPTION
**Pull request by ryanfkeepers**
---

#### Does this PR need a docs update or release note?

- [x] :no_entry: No

#### Type of change

- [x] :sunflower: Feature

#### Issue(s)

* #2757

#### Test Plan

- [x] :zap: Unit test

### CODE REVIEW

File: src/cli/utils/sharepoint_test.go [hunk 1]
```diff
@@ -180,13 +180,6 @@ func (suite *SharePointUtilsSuite) TestIncludeSharePointRestoreDataSelectors() {
 			},
 			expectIncludeLen: 2,
 		},
-		{
```
Comment (vkamra) [inline src/cli/utils/sharepoint_test.go:183]: @ryanfkeepers - why is this removed?

**Comment by vkamra** [comment]
@ryanfkeepers - LGTM. There's no PR description but assuming that's because it's a PR train. I used the commit descriptions for review. Does need lint fixes.

**Review by vkamra — APPROVED** [review]

File: src/cli/utils/sharepoint_test.go (hunk 1 above)
Comment (ryanfkeepers) [reply src/cli/utils/sharepoint_test.go:183]: It isn't actually covered within that test.  I plugged it in with the last PR, then forgot to take it out when I realized it was the wrong test.
